examples/decomposition/plot_mv_ica_tutorial.py

```diff
@@ -22,7 +22,9 @@
 import numpy as np
 import matplotlib.pyplot as plt
 
-from mvlearn.decomposition.mv_ica import MultiviewICA, PermICA, GroupICA
+from mvlearn.decomposition.multiviewica import MultiviewICA
+from mvlearn.decomposition.permica import PermICA
+from mvlearn.decomposition.groupica import GroupICA
```

Comment:
can you change these 3 imports to be from the module, rather than from the file? It just looks simpler/cleaner